src/domain/consents.ts

```diff
@@ -42,6 +42,9 @@ import { logger } from '~/shared/logger'
 import { Enum } from '@mojaloop/central-services-shared'
 import { ExternalScope, convertExternalToScope } from '../lib/scopes'
 import { DatabaseError } from './errors'
+import {
+  thirdparty as tpAPI
+} from '@mojaloop/api-snippets'
 
 export interface PostConsentPayload {
```

Comment:
should we update this interface? initiatorId & participantId are coming via headers not payload

Reply:
ok removed this interface, actually it's not required as  api-snippets used for types